lib/src/components/dropdown/dropdown.dart

```diff
@@ -0,0 +1,433 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import '../../../zeta_flutter.dart';
+
+/// Class for [ZetaDropdown]
+class ZetaDropdown extends StatefulWidget {
+  ///Constructor of [ZetaDropdown]
+  const ZetaDropdown({
+    super.key,
+    required this.items,
+    required this.onChange,
+    required this.selectedItem,
+    this.rounded = true,
+    this.checkBoxType = CheckboxType.none,
+    this.isMinimized = false,
+  });
+
+  /// Input items as list of [ZetaDropdownItem]
+  final List<ZetaDropdownItem> items;
+
+  /// Currently selected item
+  final ZetaDropdownItem selectedItem;
+
+  /// Handles changes of dropdown menu
+  final ValueSetter<ZetaDropdownItem> onChange;
+
+  /// {@macro zeta-component-rounded}
+  final bool rounded;
+
+  /// If checkbox is to be shown and if its circular or square
+  final CheckboxType checkBoxType;
```

Comment:
The rounded isnt a checkbox, it should be a radio button. Sorry I missed that first time round

Reply:
So the checkbox should only be square?

Reply:
yeah, 
you can use ZetaCheckbox for that, and RadioButton is built in (we can change to ZetaRadioButton later when it is completed)

Reply:
I've included these changes.  switched to a leading type enum to cycle through options